scripts/west_commands/blobs.py

```diff
@@ -0,0 +1,166 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import hashlib
+import importlib
+import os
+from pathlib import Path
+import re
+import sys
+import textwrap
+
+from abc import ABC, abstractmethod
+from urllib.parse import urlparse
+from west import log
+from west.commands import WestCommand
+
+from fetchers.fetcher import fetchersmeta
+from zephyr_ext_common import ZEPHYR_BASE
+
+sys.path.append(os.fspath(Path(__file__).parent.parent))
+import zephyr_module
+
+class Blobs(WestCommand):
+
+    def __init__(self):
+        super().__init__(
+            'blobs',
+            # Keep this in sync with the string in west-commands.yml.
+            'work with binary blobs',
+            'Work with binary blobs',
+            accepts_unknown_args=False)
+
+    def do_add_parser(self, parser_adder):
+        default_fmt = '{module}: {status} {path} ({type}, {abspath})'
```

Comment:
Use of colons, parentheses, and commas makes this default format string unfriendly to tools like cut and awk. Did you consider that and choose this anyway?

Reply:
I did, because this tool has an `-f/--format` param which you can use to format the output in any way you want. However, I do agree that, for consistency with `west list`, plain space-separated items would be better. I will make the change.